server/routerlicious/packages/services-shared/src/socketIoRedisConnection.ts

```diff
@@ -30,7 +30,7 @@ export class SocketIoRedisSubscriptionConnection
      */
     private readonly subscriptions: Map<string, (channel: string, messageBuffer: Buffer) => void> = new Map();
 
-    constructor(client: redis.RedisClient) {
+    constructor(client: Redis) {
         super(client);
 
```

Comment:
Do we want to add an error listener for the Redis client such as in server/routerlicious/packages/services/src/redisClientManager.ts?
```ts
client.on("error", (error) => {
            winston.error("Client Manager Redis Error:", error);
        });
```